luigi/contrib/sge.py

```diff
@@ -203,6 +203,7 @@ class SGEJobTask(luigi.Task):
         description="don't tarball (and extract) the luigi project files")
 
     def __init__(self, *args, **kwargs):
+        super(SGEJobTask, self).__init__(*args, **kwargs)
```

Comment:
@Tarrasch this is another change you need to pay attention, it seems `SGEJobTask` is not calling `super.__init__` which break the test case some how.